UI/data/locale/en-US.ini

```diff
@@ -1225,6 +1230,9 @@ YouTube.Actions.Error.NoBroadcastCreated="Broadcast creation error '%1'.<br/>A d
 YouTube.Actions.Error.NoStreamCreated="No stream created. Please relink your account."
 YouTube.Actions.Error.YouTubeApi="YouTube API Error. Please see the log file for more information."
 YouTube.Actions.Error.BroadcastNotFound="The selected broadcast was not found."
+YouTube.Actions.Error.FileNotExists="Selected file does not exist."
```

Comment:
s/FileNotExists/FileNotExist/g

"The file does not exist."
"Selected file does not exist."

For consistency with the string variable name, I'd use "Exist" instead of "Exists".

Reply:
I changed it to `FileMissing` because that sounds more sensible.